zuul.d/layout.yaml

```diff
@@ -1,15 +1,23 @@
 ---
 # zuul.d/layout.yaml
+- job:
+    name: ansible-tox-devel
+    parent: ansible-tox-py36
+    vars:
+      tox_envlist: devel
+
+- job:
```

Comment:
what you need to do here, is add yourself a pre-run playbook, to ansible playbook that includes the install-docker role from zuul jobs.